content/pt-br/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,528 @@
+
+O tipo específico de parâmetros a serem usados depende do controlador Ingress que você especificar no campo `.spec.controller` da classe Ingress.
+
+### Escopo da classe Ingress
+
+Dependendo do seu controlador Ingress, os parâmetros definidos em todo o cluster ou apenas para um namespace poderão ser utilizados.
+
+{{< tabs name="tabs_ingressclass_parameter_scope" >}}
+{{% tab name="Cluster" %}}
+O escopo padrão para os parâmetros da classe Ingress é em todo o cluster.
+
+Se você definir o campo `.spec.parameters` e não definir `.spec.parameters.scope`, ou se você definir `.spec.parameters.scope` como Cluster, então a classe Ingress se refere a um recurso com escopo de cluster. 
+O `kind` (em combinação com o `apiGroup`) dos parâmetros refere-se a uma API com escopo de cluster (possivelmente um recurso personalizado), e o `name` dos parâmetros identifica um recurso específico com escopo de cluster para essa API.
+
+Por exemplo:
+
+```yaml
+---
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  name: external-lb-1
+spec:
+  controller: example.com/ingress-controller
+  parameters:
+    # Os parâmetros para esta classe Ingress são especificados em um
+    # ClusterIngressParameter (grupo de API k8s.example.net) nomeado
+    # "external-config-1". Esta definição diz ao Kubernetes para
+    # procurar um recurso de parâmetro com escopo de cluster.    
+    scope: Cluster
+    apiGroup: k8s.example.net
+    kind: ClusterIngressParameter
+    name: external-config-1
+```
+{{% /tab %}}
+{{% tab name="Namespaced" %}}
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
+Se você definir o campo `.spec.parameters` e definir `.spec.parameters.scope` como `Namespace`, a classe Ingress terá como referência um recurso com escopo de `namespace`. 
+Você também deve definir o campo namespace dentro de `.spec.parameters` para o namespace que contém os parâmetros que deseja usar.
+
+O campo `kind` (em combinação com o campo `apiGroup`) dos parâmetros refere-se a uma API com namespace (por exemplo: `ConfigMap`), e o campo `name` dos parâmetros identifica um recurso específico no namespace que você especificou no campo `namespace`.
+
+Os parâmetros com escopo de namespace ajudam o operador de cluster a delegar o controle sobre a configuração (por exemplo: configurações do balanceador de carga, definição de gateway API) que é usada para uma carga de trabalho. 
+Se você usou um parâmetro com escopo de cluster, então:
+
+- A equipe do operador do cluster precisa aprovar as alterações de uma equipe diferente toda vez que houver uma nova alteração de configuração sendo aplicada.
+- O operador de cluster deve definir controles de acesso específicos, como funções e vínculos [RBAC](/docs/reference/access-authn-authz/rbac/), que permitem que a equipe do aplicativo faça alterações no recurso de parâmetros do escopo do cluster.
+
+A própria API da classe Ingress é sempre com escopo de cluster.
```

Comment:
This phrase is confusing. What does it means? I guess it is stating probably that "IngressClass is a cluster scoped object"?